### DISCUSSION TITLE
Production.ERROR: Undefined index after update version or deploy production

### DISCUSSION THREAD

**Discussion by emkcloud** [livewire/livewire#3757] [7. Bugs (archived)] [answered]
Every time I run or deploy in production the Laravel logs get a lot of hydrate errors, as many clients still have the page open on some page of my site and when they execute a Livewire request they get this error.

vendor/livewire/livewire/src/HydrationMiddleware/PerformActionCalls.php:27

Is there a way or a technique to force a reload page when I am in this condition? Like when "$ resquest ['payload'][' id '] is undefined?

![pantalla](https://user-images.githubusercontent.com/5731295/133470825-df847f0e-dc3b-4ae2-9a33-f02050388440.png)

**Comment by dbfx** [comment]
Can confirm this for myself as well 

![image](https://user-images.githubusercontent.com/182394/133471382-e4154b35-7137-4228-b6a1-75c5b1aedc43.png)

**Comment by joshhanley** [comment, marked as answer]
@massimodellarovere and @dbfx please see this comment for a solution https://github.com/livewire/livewire/discussions/3724#discussioncomment-1332569 🙂
